tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-rocm.yml

```diff
@@ -35,7 +35,7 @@ stages:
             --build-arg DEVTOOLSET_ROOTPATH=/opt/rh/devtoolset-10/root
             --build-arg PREPEND_PATH=/opt/rh/devtoolset-10/root/usr/bin:
             --build-arg LD_LIBRARY_PATH_ARG=/opt/rh/devtoolset-10/root/usr/lib64:/opt/rh/devtoolset-10/root/usr/lib:/opt/rh/devtoolset-10/root/usr/lib64/dyninst:/opt/rh/devtoolset-10/root/usr/lib/dyninst:/usr/local/lib64:/usr/local/lib
-          Repository: onnxruntimetrainingrocmbuild-torch1.10.0-rocm5.0.1
+          Repository: onnxruntimetrainingrocmbuild-torch1.11.0-rocm5.1.1
```

Comment:
I think we can remove the torch version (but keep the rocm version) in the repository name and delete all the old repositories that have the torch version string in the repository name.

Reply:
If each rocm version corresponds to only one pytorch version in our python package pipeline, I think we can change it.